<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplashScreen</class>
 <widget class="QMainWindow" name="SplashScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QFrame" name="drop_shadow_frame">
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-color:rgb(38, 23, 66);
	color:rgb(220, 220, 220);
	border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>661</width>
         <height>101</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 120, 255);</string>
       </property>
       <property name="text">
        <string>&lt;strong&gt;MAS&lt;/strong&gt; Linea Aqua</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>661</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 170, 255);</string>
       </property>
       <property name="text">
        <string>&lt;strong&gt;AI&lt;/strong&gt; powered Ultrasonic Tagging Machine </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="newStyle">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>170</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Candara</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(255, 170, 255);
    color: #333333;
    border-radius: 20px;
    border: none;
    padding: 10px 20px;
    font-size: 17px;
	cursor: pointer;
}

QPushButton:hover {
    background-color: rgb(255, 120, 255);
	font-size: 18px;
}

QPushButton:pressed {
    background-color: rgb(255, 170, 255);
}
</string>
       </property>
       <property name="text">
        <string>Add New Style</string>
       </property>
      </widget>
      <widget class="QPushButton" name="oldStyle">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>170</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Candara</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(255, 170, 255);
    color: #333333;
    border-radius: 20px;
    border: none;
    padding: 10px 20px;
    font-size: 17px;
	transition: background-colo 0.2s ease-in-out;
}

QPushButton:hover {
    background-color: rgb(255, 120, 255);
	font-size: 18px;
}

QPushButton:pressed {
    background-color: rgb(255, 170, 255);
}
</string>
       </property>
       <property name="text">
        <string>Select Style</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>250</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semibold</family>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(128, 128, 128);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Created&lt;/span&gt;&lt;span style=&quot; font-weight:400; font-style:italic;&quot;&gt;: Linea Aqua R &amp;amp; D Team&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="quit">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>0</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Candara</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(38, 23, 66);
    color: #333333;
    border-radius: 20px;
    border: none;
    padding: 10px 20px;
    font-size: 20px;
}

</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
